src/otx/algo/action_detection/x3d_fastrcnn.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""X3DFastRCNN model implementation."""
+from __future__ import annotations
+
+from otx.algo.utils.mmconfig import read_mmconfig
+from otx.core.model.entity.action_detection import MMActionCompatibleModel
+
+
+class X3DFastRCNN(MMActionCompatibleModel):
```

Comment:
by the way, is this fastrcnn or faster rcnn?

Reply:
It is fastrcnn, it uses pre-computed proposals.

Reply:
thanks for the clarification